pkg/util/utils.go

```diff
@@ -89,7 +89,7 @@ func RelevantJob(jobName, status string, filter *regexp.Regexp) bool {
 
 func IsActiveBug(bug bugsv1.Bug) bool {
 	switch bug.Status {
-	case "VERIFIED", "RELEASE_PENDING", "CLOSED":
+	case "VERIFIED", "RELEASE_PENDING", "CLOSED", "CLOSED WONTFIX", "CLOSED DUPLICATE", "CLOSED NOTABUG":
```

Comment:
I don't think this is actually going to matter.

sippy gets its bug status information from ci-search, and ci-search mostly ignores closed bugs.  It only uses the "CLOSED" status to mark a bug that has disappeared from its queries as closed, until that bug is completely dropped from its query results:

https://github.com/openshift/ci-search/pull/98/files#diff-b1c84977729de3db2e886f505eb5b0624942a5b9a6177554782cca5269e0b728R171

So sippy should not be seeing bugs with any of these other statuses.

Reply:
(if sippy is referencing a bug, but it's "closed wontfix" in bugzilla, check ci-search.  The most likely cause is that ci-search's data is out of sync w/ bugzilla).